index.js

```diff
@@ -7,6 +7,8 @@ List.Item = ListItem
 
 var ListPrototype = List.prototype
 var ListItemPrototype = ListItem.prototype
+var IterPrototype = Iter.prototype
+var $iterator = Symbol && Symbol.iterator
```

Comment:
I think this should be `iterator = typeof Symbol === 'undefined' ? undefined : Symbol.iterator`

Reply:
Either works 👍